src/order/filter/at_top_bot.lean

```diff
@@ -516,4 +516,16 @@ lemma tendsto_at_top_of_monotone_of_subseq {ι ι' α : Type*} [preorder ι] [pr
   tendsto u at_top at_top :=
 tendsto_at_top_of_monotone_of_filter h (map_ne_bot hl) (tendsto_map' H)
 
+lemma tendsto_neg_at_top_at_bot {α : Type*} [nonempty α] [decidable_linear_ordered_add_comm_group α] : tendsto (has_neg.neg : α → α) at_top at_bot :=
```

Comment:
This line is really too long. There is a (more or less strictly enforced) 100 characters limit in mathlib, that you can see in vscode in the form of a vertical line.